HISTORY.md

```diff
@@ -11,10 +15,10 @@
 - Bumps `open-autonomy` framework to `v0.10.5.post2` #70
 
 
-## 0.10.6 (2023-05-31)
+## 0.10.7 (2023-05-31)
 
 - Bumps to `tomte@v0.2.12` and cleans up the repo #69
-- Bumps `open-aea` and `open-autonomy` frameworks to `v1.34.0` and `v0.10.5.post1` correspondingly #68
+- Bumps `open-aea` and `open-autonomy` frameworks to `v1.35.0` and `v0.10.5.post1` correspondingly #68
```

Comment:
Why this change?

Reply:
I bumped all the frameworks replicating what Ardian did on watchtower. Should I only bump open-autonomy?

Reply:
No I mean you updated the history incorrectly. This is an older version of the oracle that you edited.

Reply:
`0.10.7` needs to be `0.10.6` again as well :sweat_smile:

Reply:
done